workspaces/bulk-import/plugins/bulk-import/src/utils/repository-utils.tsx

```diff
@@ -383,7 +384,7 @@ export const convertKeyValuePairsToString = (
 ): string => {
   return keyValuePairs
     ? Object.entries(keyValuePairs)
-        .map(([key, value]) => `${key.trim()}: ${value.trim()}`)
+        .map(([key, value]) => `${key?.trim()}: ${value?.trim()}`)
```

Comment:
Should we filter keyValuePairs without key, or fall back to an empty string?

I'm worried that this will render undefined: undefined.